variables.tf

```diff
@@ -49,6 +49,13 @@ variable "workflow_config" {
 }
 
 
+variable "default_multipart_chunksize_mb" {
```

Comment:
Possibly put in alphabetical order, under buckets variable.

Reply:
Should've been under Optional. Fixed.